bokehjs/src/less/main.less

```diff
@@ -38,7 +38,8 @@
 
 .bk-tooltip.bk-left::before {
   position: absolute;
-  top: 40%;
+  margin: -6px 0 0 0;
```

Comment:
I'm not sure why `-6px` is needed instead of `-5px`, but `-5px` feels off-center. Perhaps I'm doing box model computations wrong.